test/Stream.t.sol

```diff
@@ -434,3 +429,66 @@ contract StreamTokenAndOutstandingBalanceTest is StreamTest {
         vm.stopPrank();
     }
 }
+
+contract StreamWithRemainderTest is StreamTest {
```

Comment:
Would be good to add another test with more realistic duration than 300 seconds, e.g 6 months

Reply:
thanks, added!